docker/docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+data:
+    image: busybox
+    volumes:
+        - ../:/var/www/app
+
+nginx:
+    image: yappabe/nginx
```

Comment:
Yes, official images need some additional configuration and this setup should be straight-forward for a developer. I wanted to provide an easy way to develop Sylius within containers.